packages/go_router/test/go_router_test.dart

```diff
@@ -5502,6 +5473,94 @@ void main() {
       ),
     );
   });
+
+  testWidgets('should allow route paths without leading /',
```

Comment:
not entirely sure about this test, wasn't this already allowed?

Reply:
yes but it's descriptive imo that both are allowed